docs/pages/management/security/revoking-access.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Revoking Access
+description: Learn how to revoke access before Teleport certificates expire
+---
+
+Teleport's approach to using short-lived certificates for all infrastructure
+access means that it can generate large numbers of certificates every day. For
+this reason, Teleport does not support traditional certificate revocation.
+
+There are two options available for revoking access: CA rotations, and Teleport locks.
+
+## CA Rotations
+
+To generate a new certificate authority and invalidate user certificates issued
+by the current CA, run `tctl auth rotate --type-user`. This process will require
+that the newly generated CA certificate is uploaded to your entire fleet of
+OpenSSH servers. This can be a disruptive change, especially in environments
+that lack automation, so proceed with caution.
+
```

Comment:
Since this is a conceptual guide that combines concepts we document elsewhere under a commonly searched-for term, I would add a link to the CA rotation docs from here.